vspec2x.py

```diff
@@ -199,11 +202,11 @@ def processDataTypeTree(parser: argparse.ArgumentParser, args, include_dirs,
     Helper function to process command line arguments and invoke logic for processing data
     type information provided in vspec format
     """
-    # check that both required arguments are available to process the data
-    # type tree
-    if args.types_output_file is None or args.vspec_types_file is None:
+    if args.vspec_types_file is None:
         parser.error(
             "Please provide the vspec data types file and the output file")
+    if args.types_output_file is None:
+        logging.info("Sensors and custom data types will be consolidated into one file")
```

Comment:
This needs to be reflected in documentation in some way: https://github.com/COVESA/vss-tools/blob/master/docs/vspec2x.md

Reply:
ack. once we align on the usage of the custom keyword, I'll update the documentation in one go.